### PR TITLE
add force debug feature

### PR DESCRIPTION
**Pull request by xlc**
so that I can make debugging build easier 

let me know if I should add this to system parachains as well

### CODE REVIEW

File: relay/kusama/Cargo.toml [hunk 1]
```diff
@@ -115,6 +115,8 @@ xcm = { package = "staging-xcm", default-features = false , version = "4.0.0" }
 xcm-executor = { package = "staging-xcm-executor", default-features = false , version = "4.0.0" }
 xcm-builder = { package = "staging-xcm-builder", default-features = false , version = "4.0.0" }
 
+sp-debug-derive = { default-features = false , version = "10.0.0" }
```
Comment (bkchr) [inline relay/kusama/Cargo.toml:118]: You forgot to enable the `std` feature.

File: relay/polkadot/Cargo.toml [hunk 2]
```diff
@@ -111,6 +111,8 @@ xcm = { package = "staging-xcm", default-features = false , version = "4.0.0" }
 xcm-executor = { package = "staging-xcm-executor", default-features = false , version = "4.0.0" }
 xcm-builder = { package = "staging-xcm-builder", default-features = false , version = "4.0.0" }
 
+sp-debug-derive = { default-features = false , version = "10.0.0" }
```
Comment (bkchr) [inline relay/polkadot/Cargo.toml:114]:
```suggestion
sp-debug-derive = { default-features = false, version = "10.0.0" }
```

File: relay/kusama/Cargo.toml (hunk 1 above)
Comment (bkchr) [inline relay/kusama/Cargo.toml:118]:
```suggestion
sp-debug-derive = { default-features = false, version = "10.0.0" }
```

**Review by bkchr — APPROVED** [review]

**Review by muharem — APPROVED** [review]

**Review by liamaharon — APPROVED** [review]
I think we should also add for SPs.

**Review by joepetrowski — APPROVED** [review]

**Review by acatangiu — APPROVED** [review]

**Review by ggwpez — APPROVED** [review]

**Pull request merged**